### ISSUE TITLE
Warning on renaming null and cell file: already exists

### ISSUE THREAD

**Issue by SFrav** [NikosAlexandris/i.fusion.hpf#6]
The pansharpening works, but produces the following warnings along the way:
`<WARNING: Unable to rename null file 'D:\Documents\grassdata/newLocation/.tmp/unknown/2544.1' to 'D:\Documents\grassdata/newLocation/cell_misc/tmp.5908.4_msx_hpf/null': File exists
WARNING: Unable to rename cell file 'D:\grassdata/newLocation/Morogoro/.tmp/unknown/2544.0' to 'D:\grassdata/newLocation/fcell/tmp.5908.4_msx_hpf': File exists>`

**Comment by NikosAlexandris** [comment]
Even by using the `--o`  (=overwrite)  instruction?

**Comment by SFrav** [comment]
Yes. This is on Windows, so maybe the different command lines at play.

**Comment by NikosAlexandris** [comment]
Maybe a "typical" path naming problem?  The path `D:\Documents\grassdata/newLocation/.tmp/unknown/2544.1`  has two backslashes (which is normal for Windows, as far as I can remember) and then slashes.  Is this correct?

**Comment by SFrav** [comment]
That's right.

**Comment by SFrav** [comment]
There could be more backslashes depending on where the working directory is located.

**Comment by pmav99** [comment]
Wouldn't [normpath](https://docs.python.org/2/library/os.path.html#os.path.normpath) help?

**Comment by NikosAlexandris** [comment]
Yes, this looks nice! We need a Windows user. And an edit of the respective line in the script. Thanks!

**Comment by SFrav** [comment]
Happy to test. Just let me know what line needs to be added to which file.

**Comment by pmav99** [comment]
@NikosAlexandris 
I guess it is safe enough to add a `normpath` call even without explicitly testing it ;)

@SFrav 
It would be better to wait for Nikos' comment but I think that it is needed whenever `grass.tempfile()` is called. BTW, since you are on windows, could you run on the grass console the following command:
```
g.tempfile -d pid=61322
```
If this command returns a name with mixed slashes then we know where the original bug is.

**Comment by NikosAlexandris** [comment]
@pmav99 Let us, please, stick to the testing practice. It's a learning process as well and a good habit. If it works, let's pull it in :+1:
